Modules/Core/Sources/Core/Buffer/Buffer.swift

```diff
@@ -1,10 +1,11 @@
-@_exported import struct Dispatch.DispatchData
 #if os(Linux)
-import Glibc
+    import Glibc
 #else
-import Darwin
+    import Darwin
 #endif
 
+@_exported import struct Dispatch.DispatchData
```

Comment:
The import - we already have a `public typealias Buffer = DispatchData` which acts like an export.